changelog/split-std-datetime.dd

```diff
@@ -0,0 +1,43 @@
+std.datetime has been split into a package.
+
+std.datetime is now a package containing the following modules:
+
+$(UL
+    $(LI <a href="$(PHOBOS_PATH)std_datetime_common.html">std.datetime.common</a>)
+    $(LI <a href="$(PHOBOS_PATH)std_datetime_date.html">std.datetime.date</a>)
+    $(LI <a href="$(PHOBOS_PATH)std_datetime_datetime.html">std.datetime.datetime</a>)
+    $(LI <a href="$(PHOBOS_PATH)std_datetime_interval.html">std.datetime.interval</a>)
+    $(LI <a href="$(PHOBOS_PATH)std_datetime_systime.html">std.datetime.systime</a>)
+    $(LI <a href="$(PHOBOS_PATH)std_datetime_timeofday.html">std.datetime.timeofday</a>)
+    $(LI <a href="$(PHOBOS_PATH)std_datetime_timezone.html">std.datetime.timezone</a>)
+)
+
+$(LI <a href="$(PHOBOS_PATH)std_datetime.html">std.datetime.package</a>)
```

Comment:
This seems very incorrect. Don't use html `<a>` tags here. Confession: I don't know the right way. Just look at other changlog PRs.